Source/IncrementableLabel.swift

```diff
@@ -150,6 +183,10 @@ extension IncrementableLabel {
                 text = String(format: format, currentValue)
             }
         }
+        //Update the color
+        if startColor != nil && endColor != nil{
```

Comment:
could you unwrap these optionals in the if instead of force unwrap it after?

```
if let startColor = startColor, let endColor = endColor {
            textColor = blend(from: startColor, to: endColor, percent: progress / duration)
}
```